swagger_ui/core.py

```diff
@@ -142,13 +140,10 @@ def get_config(self, host):
                 config = _load_config(config_file.read())
         elif self.config_spec:
             config = _load_config(self.config_spec)
+        else:
+            raise RuntimeError('No config found!')
 
-        version = config.get('openapi', '2.0.0')
-        if Version(version) >= Version('3.0.0'):
```

Comment:
Why delete these? I remember it use `servers` in openapi version 3 rather than `host`.

Reply:
It used host rather than servers, it overrode every URL entry in servers with host, which is a behavior not defined at all in OpenAPI, but more importantly it prevented the standard use case of just setting a URL. If the old behavior is useful in someway, maybe it could be enabled via a configuration option?